### PR TITLE
Allow test failures to be reported to the CLI

### PR DESCRIPTION
**Pull request by rjkip**
https://www.pivotaltracker.com/story/show/104880456

### CODE REVIEW

**Comment by rjkip** [comment]
Please review @DRvanR, @arothuis

File: src/Surfnet/Conext/EntityVerificationFramework/SuiteResult.php [hunk 1]
```diff
@@ -54,12 +53,12 @@ public static function success()
         return new self(self::STATUS_SUCCESS);
     }
 
-    public static function failedTest(VerificationTestResult $verificationTestResult, VerificationTest $failedTest)
+    public static function failedTest(VerificationTestResult $verificationTestResult, $failedTestName)
     {
         $result = new self(self::STATUS_FAILED);
```
Comment (DRvanR) [inline src/Surfnet/Conext/EntityVerificationFramework/SuiteResult.php]: please add `Assert::string` and `Assert::notEmpty` for `$failedTestName`

File: src/Surfnet/Conext/OperationsSupportBundle/Tests/Reporter/CliReporterTest.php [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+final class CliReporterTest extends TestCase
+{
+    /**
+     * @test
+     * @group reporter
+     */
+    public function it_writes_a_report_to_output()
+    {
+        $output = m::mock(OutputInterface::class);
+        $output->shouldReceive('writeln');
```
Comment (DRvanR) [inline src/Surfnet/Conext/OperationsSupportBundle/Tests/Reporter/CliReporterTest.php]: `->once()`?

File: src/Surfnet/Conext/OperationsSupportBundle/Tests/Reporter/CliReporterTest.php [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+            )
+        );
+    }
+
+    /**
+     * @test
+     * @group reporter
+     * @expectedException \Surfnet\Conext\OperationsSupportBundle\Exception\LogicException
+     */
+    public function it_should_thrown_when_asked_to_write_a_report_of_a_succeeded_test()
```
Comment (DRvanR) [inline src/Surfnet/Conext/OperationsSupportBundle/Tests/Reporter/CliReporterTest.php]: it should thrown when asked? You wot? :tongue: 

as an aside: personally I'd prefer `an_exception_is_thrown_when_reporting_a_successful_result` (and likewise `failed_reports_are_written_to_output`)

File: src/Surfnet/Conext/OperationsSupportBundle/Tests/Reporter/CliReporterTest.php (hunk 2 above)
Comment (rjkip) [inline src/Surfnet/Conext/OperationsSupportBundle/Tests/Reporter/CliReporterTest.php]: `->atLeastOnce()`?

Comment (DRvanR) [inline src/Surfnet/Conext/OperationsSupportBundle/Tests/Reporter/CliReporterTest.php]: fair enough

**Comment by rjkip** [comment]
Oi, maties @DRvanR, @arothuis. Rebased on develop and added only a8e6373...84e0b5a.

**Comment by DRvanR** [comment]
:+1:

**Comment by rjkip** [comment]
@arothuis :+1:?

**Comment by arothuis** [comment]
:+1:

**Pull request merged**